helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -34,8 +39,6 @@ spec:
             imagePullPolicy: {{ .Values.image.pullPolicy }}
             resources: {{- toYaml .Values.head.resources | nindent 14 }}
             env:
-              - name: TYPE
```

Comment:
I did not find any use of the `TYPE` environment variable, but is it possible for Ray image to use the environment variable?

Reply:
I don't think so.

Reply:
Got it.